Chamasma/Chamasma/View/CategoryView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CategoryView.swift
+//  Chamasma
+//
+//  Created by Mariam Ormotsadze on 22.04.23.
+//
+
+import DesignSystem
+import SwiftUI
+
+public struct CategoryView: View {
+    // MARK: - Properties
+    public let title: String
+    // TODO: pass [CategoryItemModel] when MealCell will be refactored to take model not properties
+    // MARK: - Body
+    public var body: some View {
+        ScrollView {
+            VStack {
+                header
+                categoryItemList
+            }
+        }
+    }
+    private var header: some View {
+        VStack {
+            imageHeader
+            ListSectionHeader(title: title)
+        }
+    }
+    private var imageHeader: some View {
+        DesignSystem.Image.burger
+            .resizable()
+            .frame(height: 160)
+            .cornerRadius(8)
+    }
+    private var categoryItemList: some View {
+        VStack(alignment: .leading) {
+            ForEach((1...10), id: \.self) { _ in // TODO: will iterate given array
+                MealCell(
+                    name: "Cheesburger",
+                    description: "Cheesburger, Regular Bun, ketchup, pickle",
+                    price: 123.45,
+                    rating: 0.78,
+                    image: DesignSystem.Image.burger,
+                    isPopular: true
+                )
```

Comment:
For the current and future cases, please write `example` for each such component in extension and then use that `example` object for any needs

Reply:
I don't see why would I change it right now, As. I commented on this PR CategoryView needs to accept array of models, it will change anyway so I am not going to waste time to making extension, just to erase it after 1 hour

Reply:
Mariam, first of all, please do not be nervous.

I want to have a style in the project to not write custom creations of some objects. Instead, it would be better to write one extension `example` of each struct and then use it wherever we need it.

So the code is cleaner.

In any case like this, you should use that style.

If you are going to change it after that PR, of course, you do not need to change it now.

But, please, do not forget to be polite in the conversations.

Reply:
Btw, you could have used MenyCategory.example for that case

Reply:
I am nor nervous, neither impolite
I agree that you want to have a style in the project, appreciate it and fully share that intention, just made sure that you read my comment and changing it now was not relevant, thats all

Reply:
I could not find that, will take into consideration, thanks

Reply:
"so I am not going to waste time to" is not a polite way to say something at all

Reply:
how would you say that politely then?

Reply:
"Since this is a very temporary code and I am going to change it in the next PR, it is not mandatory to change it now, right? Do you agree?"

Reply:
It is literally 01:41 and I have two brain cells left, sorry that my comment seemed impolite, I did not have that intentions